src/analysis/seminorm.lean

```diff
@@ -237,6 +244,29 @@ instance zero_hom_class : zero_hom_class (add_group_seminorm E) E ℝ :=
 /- TODO: All the following ought to be automated using `to_additive`. The problem is that it doesn't
 see that `has_smul R ℝ` should be fixed because `ℝ` is fixed. -/
 
+variable (E)
+
+/-- The trivial norm on an additive group `E` is the `add_group_seminorm` taking value `0` at `0`
+  and `1` at every other element. -/
+def trivial_norm [decidable_eq E] : add_group_seminorm E :=
```

Comment:
Can you multiplicativise this?

Reply:
Now in #16237

Reply:
I'm looking at #16227 now; I'll check #16237 next.
Can I merge this if it passes CI?

Reply:
Does/should this PR depend on those other PRs?

Reply:
I understand that it would be nice to get all of these merged asap. If you can tell me in which order they are supposed to be merged, that would be helpful.
(And sorry for the delays in reviews.)

Reply:
I think this one can be merged independently of others if you remove `trivial_norm` from it. The greater picture is [on Zulip](https://leanprover.zulipchat.com/#narrow/stream/113488-general/topic/Reorganising.20norms).

Reply:
Why is `trivial_norm` an issue?

Reply:
Simply because it appears in #16237 as well.

Reply:
Indeed. Given that #16237 has just been unlocked, another option is to wait for it to go through.

Reply:
Yes, I think at this point that's probably best.